addons/common/functions/fnc_progressBar.sqf

```diff
@@ -30,8 +30,12 @@ private _player = ACE_player;
 closeDialog 0;
 createDialog QGVAR(ProgressBar_Dialog);
 
-// Hide cursor by using custom transparent cursor
 private _display = uiNamespace getVariable QGVAR(dlgProgress);
+
+// Ensure CBA keybindings are hooked into the display
+[_display] call (uiNamespace getVariable ["CBA_events_fnc_initDisplayCurator", {}]);
```

Comment:
Is there a point in having fallback value? ACE can't be used without CBA anyway.
```suggestion
_display call (uiNamespace getVariable "CBA_events_fnc_initDisplayCurator");
```

Reply:
Just a matter of habit.